test/unittests/libethereum/ValidationSchemes.cpp

```diff
@@ -0,0 +1,192 @@
+/*
+    This file is part of cpp-ethereum.
```

Comment:
Please change the note to this one: https://github.com/ethereum/aleth/issues/5403.